addon/mixins/frost-list-sorting-mixin.js

```diff
@@ -1,37 +1,32 @@
 import Ember from 'ember'
 const {
   Mixin,
-  on
+  get,
+  on,
+  set,
+  defineProperty,
+  computed: {alias}
 } = Ember
-import computed from 'ember-computed-decorators'
+import {normalizeSort, defaultSort} from 'ember-frost-list/utils/utils'
```

Comment:
Can we re-export these utils from the index.js and avoid the path?  If they aren't intended to be accessed externally then you can ignore this.

I might also suggest that instead of one generic utils module you would call this `utils/sort` since it contains sort utils